### PR TITLE
UI: deterministic uuid generation in UI tests

### PR DESCRIPTION
**Pull request by DimaAmega**

### CODE REVIEW

File: src/evidently/ui/demo_projects/bikes.py [hunk 1]
```diff
@@ -89,6 +89,8 @@ def create_data():
 def create_report(i: int, data):
     current, reference, column_mapping = data
 
+    # for deterministic tags generation
+    random.seed(i + 123)
```
Comment (svetleo-evidently) [inline src/evidently/ui/demo_projects/bikes.py:93]: Better as a constant in model level

File: src/evidently/cli/ui.py [hunk 2]
```diff
@@ -7,11 +7,26 @@
...
+
+    uuid.uuid4 = deterministic_uuid4
+
+
 @app.command("ui")
 def ui(
     host: str = Option("0.0.0.0", help="Service host"),
     port: int = Option(8000, help="Service port"),
     workspace: str = Option("workspace", help="Path to workspace"),
+    use_deterministic_uuid: bool = Option(False, help="is use deterministic uuid generation"),
```
Comment (svetleo-evidently) [inline src/evidently/cli/ui.py:29]: Looks like mistype
help="use deterministic uuid generation or not"

File: src/evidently/ui/demo_projects/bikes.py (hunk 1 above)
Comment (DimaAmega) [reply src/evidently/ui/demo_projects/bikes.py]: Yep! Fixed it

**Review by svetleo-evidently — APPROVED** [review]

File: src/evidently/cli/ui.py [hunk 3]
```diff
@@ -20,6 +35,9 @@ def ui(
     secret: Optional[str] = Option(None, help="Secret for writing operations"),
 ):
     """Start Evidently UI service"""
+    if os.environ.get("EXPERIMENTAL_DETERMINISTIC_UUID"):
```
Comment (svetleo-evidently) [inline src/evidently/cli/ui.py:38]: 👍

**Pull request merged**